lib/pages/route/route_list.dart

```diff
@@ -232,17 +241,20 @@ class _ListadoRutasScreenState extends State<ListadoRutasScreen> {
         itemCount: rutasFiltradas!.length,
         itemBuilder: (context, index) {
           final ruta = rutasFiltradas![index]; // Obtiene la ruta actual.
+          final esLocal = ruta.containsKey('local') && ruta['local'] == 1;  // Verifica si la ruta es local
+          final cardColor = esLocal ? Colors.orange[50] : Colors.green[50]; // Define el color del Card basado en el estado // Define el color basado en el estado
 
           return Card(
             margin: const EdgeInsets.all(8.0),
             shape: RoundedRectangleBorder(
               borderRadius: BorderRadius.circular(10.0),
             ),
             elevation: 5,
+            color: cardColor,
```

Comment:
Queda mejor en verde claro las locales y en null (color por defecto, blanco) para las otras
```suggestion
            color: esLocal ? Colors.green.shade50 : null,
```